src/sentry/static/sentry/app/stores/pluginsStore.tsx

```diff
@@ -82,13 +106,13 @@ const PluginsStore = Reflux.createStore({
     this.triggerState();
   },
 
-  onUpdateSuccess(id, _updateObj) {
+  onUpdateSuccess(id: string, _updateObj: Partial<Plugin>) {
```

Comment:
I noticed that we are usually defining the function types in an interface. Example:

```
type PluginStoreInterface = Reflux.StoreDefinition & {
  onUpdateSuccess: (id: string, _updateObj: Partial<Plugin>) => void;
  ...
 }
```

maybe we should do that here too?

Reply:
@priscilawebdev will the typings get used anywhere? I don't really see the point of adding types for functions if it doesn't impact type checking at all which it doesn't seem to here.

Reply:
yes you're right.. I don't know why we're creating a separate interface... Good point 😉